app.js

```diff
@@ -0,0 +1,50 @@
+const bookstorage = JSON.parse(localStorage.getItem('books-display')) || [];
+const bookname = document.getElementById('book-name');
+const authorname = document.getElementById('author-name');
+const addbutton = document.querySelector('.add-book-button');
+const presentbooks = document.querySelector('.books-display');
+
+const bookobject = {
+  title: this.title,
+  author: this.author,
+  id: this.id,
+};
+
+function addbook(title, author) {
+  const newbook = Object.create(bookobject);
+  newbook.title = title;
+  newbook.author = author;
+  newbook.id = Math.floor(Math.random() * 10);
```

Comment:
- [ ] I have noticed that you are adding the random `id` on every book created. Please ensure that the id created is unique starting from `1` to infinite. This is to ensure that the right book is deleted or edited. Also make sure that the id order is updated when a book is deleted ☺️.